package.json

```diff
@@ -51,6 +51,7 @@
     "nyc": "^15.1.0",
     "prettier": "^2.4.1",
     "pretty-quick": "^3.1.0",
+    "sfdx-cli": "^7.160.0",
```

Comment:
A dev-dep needed for `VersionDetail` typing.  It's not yet used until a version of sfdx-cli is published that exports `VersionDetail`